packages/cloud_firestore/cloud_firestore/lib/src/firestore.dart

```diff
@@ -217,10 +206,12 @@ class FirebaseFirestore extends FirebasePluginPlatform {
   }
 
   @override
-  bool operator ==(dynamic o) =>
-      o is FirebaseFirestore && o.app.name == app.name;
+  // ignore: avoid_equals_and_hash_code_on_mutable_classes
```

Comment:
Do we _really_ need those == overrides?

Reply:
That's a good question. I wasn't sure on changing API behaviour. What do you think, @Salakar?

Reply:
@rrousselGit is your thinking here that we should rely on the hashCode override instead?